python/ray/worker.py

```diff
@@ -1296,6 +1305,9 @@ def init(redis_address=None, node_ip_address=None, object_id_seed=None,
             be configured with.
         num_gpus (int): Number of gpus the user wishes all local schedulers to
             be configured with.
+        num_custom_resource (int): The quantity of a user-defined custom
+            resource that the local scheduler should be configured with. This
+            flag is highly unstable and should not be used.
```

Comment:
"will be removed" sounds like that it has been deprecated. I hope it will not just be removed, but maybe matured into something else.

Maybe just saying "experimental feature subject to changes in the future". Kubernetes has this for GPU support: https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/